front-end/src/environments/environment.prod.ts

```diff
@@ -1,6 +1,6 @@
 export const environment = {
   production: true,
-  base_api_url: '/api/',
+  base_api_url: 'https://play.osustuff.org/api/',
```

Comment:
Blegh. I *still* don't like the way this works, months later. Is there a better way I can do this?